include/private/config/wrapper/socket.h

```diff
@@ -41,4 +41,14 @@
 #    define config_get_local_socket_name no_abstract_socket_names_get_local_socket_name
 #  endif
 
+#  ifdef HAVE_DISALLOW_SIGNAL_DURING_SENDING
+#    if defined(__APPLE__)
```

Comment:
Change this condition to use `HAVE_SO_NOSIGPIPE` and the next to `HAVE_MSG_NOSIGGNAL`.